.github/workflows/build_tests.yml

```diff
@@ -71,37 +71,56 @@ jobs:
         pip install -U "sklearn"
     - name: Install POT
       run: |
-        pip install -e .
+        pip install --verbose -e .
     - name: Run tests
       run: |
         python -m pytest -v test/ ot/ --ignore ot/gpu/
 
 
   macos:
-     runs-on: macOS-latest
-     strategy:
-       max-parallel: 4
-       matrix:
-         python-version: [3.7]
-
-     steps:
-     - uses: actions/checkout@v1
-     - name: Set up Python ${{ matrix.python-version }}
-       uses: actions/setup-python@v1
-       with:
-         python-version: ${{ matrix.python-version }}
-     - name: Install dependencies
-       run: |
-         python -m pip install --upgrade pip
-         pip install -r requirements.txt
-         pip install pytest "pytest-cov<2.6"
-         pip install -U "sklearn"
-     - name: Install POT
-       run: |
-         pip install -e .
-     - name: Run tests
-       run: |
-         python -m pytest -v test/ ot/ --doctest-modules --ignore ot/gpu/ --cov=ot
+    runs-on: macOS-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.7]
+
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        # install OpenMP (via homebrew)
+        /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
+        brew install libomp
```

Comment:
can you also keep a macos build without openmp so we can check it works on a fresh machine too? thx

Reply:
yes indeed i would keep the classical build on macosx and add a build/test with openmp

Reply:
Yes, I was planning on doing this (just after I fixed the issues with the full build)